### ISSUE TITLE
implementation ('org.jitsi.react:jitsi-meet-sdk:+') { transitive = true }

### ISSUE THREAD

**Issue by gauravgupta566** [jitsi/jitsi-meet#3392]
*This Issue tracker is only for reporting bugs and tracking code related issues.*

Before posting, please make sure you check community.jitsi.org to see if the same or similar bugs have already been discussed. General questions, installation help, and feature requests can also be posted to community.jitsi.org.

## Description
---

## Current behavior
---

## Expected Behavior
---

## Possible Solution
---

## Steps to reproduce
---

# Environment details
---

**Comment by saghul** [comment]
Have you added our maven repo?

**Comment by saghul** [comment]
Can you validate if the directory you specified has been filled with files after doing the publish?

Also, are you using Windows? We don't really support it and if anything doesn't work as expected I'm afraid we won't be able to help you much, due to our lack of experience.

**Comment by gauravgupta566** [comment]
Yes, I am using window
and after publish directory 
![issue6](https://user-images.githubusercontent.com/18099872/44505349-95155380-a6be-11e8-80f0-dee8d9a3f15a.PNG)
specified has been filled

**Comment by gauravgupta566** [comment]
hi sir, if I don't write "  // implementation ('org.jitsi.react:jitsi-meet-sdk:+') { transitive = true }" in build.gradle
and run the project then it shows
![issue8](https://user-images.githubusercontent.com/18099872/44505880-5fbe3500-a6c1-11e8-8d54-b433520b50be.PNG)

then after i click on reload button at the bottom then it shows
![issue9](https://user-images.githubusercontent.com/18099872/44505897-76648c00-a6c1-11e8-9f15-4f49ab58df7d.PNG)

**Comment by gauravgupta566** [comment]
hi sir, it runs
![run](https://user-images.githubusercontent.com/18099872/44516022-9b68f700-a6e1-11e8-9959-a01920664e60.PNG)

but when i try to enter room name it gives error failed to load config.js from in android >>E/ReactNativeJS: 'Failed to load config from https://meet.jit.si/config.js?room=umenit' 

how to change url

**Comment by FaizalMalik** [comment]
@saghul  - I am also facing this issue - I am building the Standalone application for android - if use pre-build SDK artifacts/binaries it works fine .

 but If i try **Build and use your own SDK artifacts/binaries** method - I end up with 

Failed to resolve: org.jitsi.react:jitsi-meet-sdk:+

**Comment by saghul** [comment]
@FaizalMalik did you publish your artifacts to a local maven repo? Also, if you are just building Jitsi Meet, you don't need to build and publish all components yourself, you can just modify the code and run `react-native run-android`.

**Comment by gauravgupta566** [comment]
I have not added org.jitsi.react:jitsi-meet-sdk:+ then run the command it works

**Comment by gauravgupta566** [comment]
yes , it publish in local maven repo

**Comment by Dmxy** [comment]
I have a same question 
when I publish all components in the local repo, I also got "ERROR: Failed to resolve: org.jitsi.react: jitsi-meets-sdk :+".
my local repo like this:
![local_repo](https://user-images.githubusercontent.com/9995805/55546030-07442100-5701-11e9-80a6-5c5e0043cc61.png)

and the publish result:
![grade_setting](https://user-images.githubusercontent.com/9995805/55546313-9b15ed00-5701-11e9-8598-4169a9325d70.png)


there is my grade setting:
![grade_setting](https://user-images.githubusercontent.com/9995805/55546138-39ee1980-5701-11e9-879c-07108555ad8d.png)

and when I use the pre-SDK, it works.
So could you give me some advice ?

**Comment by saghul** [comment]
What is the problem? The build was succesful.

**Comment by Dmxy** [comment]
If I use local repo in grade, it will prompt me like this:
![error](https://user-images.githubusercontent.com/9995805/55693974-78772300-59e4-11e9-9868-7e0914b27427.png)

**Comment by saghul** [comment]
Use the absolute path.

**Comment by Dmxy** [comment]
it's does't work for me..:
![1234](https://user-images.githubusercontent.com/9995805/55768557-e9cfd800-5aaf-11e9-9510-c7d456fe407d.png)

**Comment by saghul** [comment]
Does it work when you use our maven repo?

**Comment by Dmxy** [comment]
Yes, when I use  maven {url "https://github.com/jitsi/jitsi-maven-repository/raw/master/releases"} to replace  maven {url "file: tmp/local_repo"}, it's all right.

**Comment by saghul** [comment]
What files do you have in /Users/xxx/Desktop/app/platforms/tmp/local_repo ?

**Comment by Dmxy** [comment]
It looks like this:
![local_repo](https://user-images.githubusercontent.com/9995805/55783679-88c0f800-5ae1-11e9-8dec-bc39a684ba28.png)

**Comment by saghul** [comment]
You are missing the Jitsi Meed SDK dependencies there. Follow the readme closely, it's documentened: https://github.com/jitsi/jitsi-meet/blob/master/android/README.md

**Comment by Dmxy** [comment]
Okay, I saw about the update. Thank you. I'll try later.

**Comment by Dmxy** [comment]
I'm sorry to consult you again, I downloaded the latest code, but in the '. / android/scripts/release - SDK. Sh/TMP/repo 'got the result:
. / android/scripts/release - SDK. Sh: line 8: realpath: command not found
. / android/scripts/release - SDK. Sh: line 10: jq: command not found
Do I need to install any tools to execute it?

**Comment by Rana-Mohit** [comment]
Hi,
I am making my own SDK artifacts/binaries.By following these steps:-

1 - Navigate to the newly added jitsi-meet folder and execute npm install after that remove video mute button from toolbox.js.
2- Create a new folder( /tmp/repo ) inside root folder
3- Copy the react native and JavaScriptCore dependency via
 cp -r node_modules/react-native/android/com tmp/repo/
 and cp -r node_modules/jsc-android/dist/org tmp/repo/
4- Run this command ./android/scripts/release-sdk.sh /tmp/repo
5- Create a new android Project
6- Copy the /tmp/repo folder into this project
7- Add maven { url “file:/tmp/repo” } in the build.gradle (Project:MyProject)
8- Add implementation (‘org.jitsi.react:jitsi-meet-sdk:+’) { transitive = true } to the build.gradle

But when i implement point 7 and 8 in android project then getting this error :-
ERROR: Failed to resolve: org.jitsi.react:jitsi-meet-sdk:+
Show in Project Structure dialog
Affected Modules: app

Note- Using stable/jitsi-meet-4468

Note - when using
maven {
            url "https://github.com/jitsi/jitsi-maven-repository/raw/master/releases"
        }
 it works perfectly.

I am missing something or did something wrong? Please help.
![Screenshot 2020-05-11 at 8 32 17 PM](https://user-images.githubusercontent.com/64829661/81576995-8d33a080-93c6-11ea-9287-a22dc6cdef44.png)

**Comment by contactedme** [comment]
Please let someone help me to solve this problem
Failed to resolve: org.jitsi.react:jitsi-meet-sdk:2.+
Show in Project Structure dialog
Affected Modules: app

**Comment by MahmudHasanMenon** [comment]
I am also facing this issue in my react-native project when i use jitsi-meet Custom SDK on Android.
![Screen Shot 2022-02-09 at 6 19 12 pm](https://user-images.githubusercontent.com/29507500/153200070-c2bd8903-aa19-4e5f-bc34-855bf3f71016.png)

Someone please help me how to resolve the issue.

**Comment by Codzaa** [comment]
I changed frameworks from Flutter to Vanilla Android Studio, but I am still getting the same error,,,,

https://user-images.githubusercontent.com/29507500/153200070-c2bd8903-aa19-4e5f-bc34-855bf3f71016.png

**Comment by Gaurav-CareMonitor** [comment]
Same error here with Mac m2
